src/neptune/metadata_containers/run_state.py

```diff
@@ -0,0 +1,45 @@
+#
```

Comment:
How about moving it directly to `metadata_containers_table` or to `internal` subpackage?

Reply:
+1